pro.vpup.vpuppr.Extension.mouse-tracker.yml

```diff
@@ -0,0 +1,19 @@
+id: pro.vpup.vpuppr.Extension.mouse-tracker
+branch: stable
+runtime: pro.vpup.vpuppr
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+appstream-compose: false
+modules:
+  - name: mouse-tracker
+    buildsystem: simple
+    build-commands:
+      - ls -a
+        #post-install:
+        #- install -Dm644 -t ${FLATPAK_DEST}/share/metainfo data/${FLATPAK_ID}.metainfo.xml
+        #- appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
```

Comment:
so the package is empty.

Reply:
For now, yes. I want to figure out why the build fails first. I should've marked this as draft, since not everything is in place yet. After I figure out why builds fail, everything else should be easy to do.